web/src/routes/login/+page.svelte

```diff
@@ -3,7 +3,7 @@
   import { CALLBACK_URL, CLIENT_ID, createAuth } from "$lib/auth";
   import { SESSION_STORAGE } from "$lib/browser_storage";
 
-  const url = new URL("http://localhost:5173/api/oauth2/authorize");
+  const url = new URL("http://localhost:3000/api/oauth2/authorize");
```

Comment:
I'd like to revert this change and the next one because i'd rather keep the proxy